app/services/security/__test__/nonce.server.test.ts

```diff
@@ -0,0 +1,15 @@
+import { generateNonce } from "../nonce.server";
```

Comment:
`it` import is missing within VSCode...
```suggestion
import { it } from "vitest";
import { generateNonce } from "../nonce.server";
```